admin/zabbix/Makefile

```diff
@@ -222,16 +230,30 @@ define Package/zabbix-server/install
 	$(call Package/zabbix/install/etc,$(1),server)
 endef
 
+define Package/zabbix-server-frontend/install
+	$(INSTALL_DIR) $(1)/www/zabbix
+	$(CP) $(PKG_BUILD_DIR)/frontends/php/* $(1)/www/zabbix
+endef
+
 define Package/zabbix-proxy/install
 	$(call Package/zabbix/install/sbin,$(1),proxy)
 	$(call Package/zabbix/install/etc,$(1),proxy)
 endef
 
+define Package/zabbix-server-frontend/postinst
+#!/bin/sh
+[ -n "$${IPKG_INSTROOT}" ] || {
+	ln -sf /www/ /www/zabbix/
+}
```

Comment:
this symlink, if really needed, should be part of the package files and not a postinst script
```
# ls -lia /www/zabbix/www
171 lrwxrwxrwx    1 root     root             5 Oct  1 04:47 /www/zabbix/www -> /www/
```

Reply:
Tested, it is not required